src/tests/ftest/server/metadata.py

```diff
@@ -16,21 +15,22 @@
 from thread_manager import ThreadManager
 
 
-def run_ior_loop(manager, uuids):
-    """IOR run for each UUID provided.
+def run_ior_loop(test, manager, num_run):
```

Comment:
Is test used?
```suggestion
def run_ior_loop(manager, num_run):
```

Reply:
Yes. `test.label_generator` because this is a thread function. Though, I forgot to pass this... Updated :)